code/model/modelread.cpp

```diff
@@ -1232,13 +1262,24 @@ int read_model_file(polymodel * pm, char *filename, int n_subsystems, model_subs
 					pm->submodel[n].look_at_num = -1; // No look_at
 				}
 
-				if ( ( p = strstr(props, "$dumb_rotate:") ) != NULL ) {
-					pm->submodel[n].movement_type = MSS_FLAG_DUM_ROTATES;
+				// note, this should come BEFORE do_new_subsystem() for proper error handling (to avoid both rotating and dumb-rotating submodel)
+				if ( ( p = strstr(props, "$dumb_rotate") ) != NULL ) {
+					pm->submodel[n].movement_type = MOVEMENT_TYPE_DUMB_ROTATE;
 					pm->submodel[n].dumb_turn_rate = (float)atof(p+13);
+
+					pm->flags |= PM_FLAG_HAS_DUMB_ROTATE;
 				} else {
 					pm->submodel[n].dumb_turn_rate = 0.0f;
 				}
 
+				// Sets can_move on submodels which are of a rotating type or which have such a parent somewhere down the hierarchy
+				if ((pm->submodel[n].movement_type != MOVEMENT_TYPE_NONE)
+					|| strstr(props, "$triggered:") || strstr(props, "$rotate") || strstr(props, "$gun_rotation:") || strstr(props, "$gun_rotation")) {
```

Comment:
As noted earlier in the look_at PR, checking for both "$gun_rotation:" and "$gun_rotation" is redundant; might as well get rid of the former while we're at it.